packages/widget/src/config.ts

```diff
@@ -26,5 +26,6 @@ export const sanitizeConfig = (config: unknown): Partial<ChatConfig> & Pick<Chat
         ...(typeof user.image === 'string' && { image: user.image }),
       },
     }),
+    ...(isObject(assistant) && ({ assistant } as Partial<Pick<ChatConfig, 'assistant'>>)),
```

Comment:
we now allow users to add an `assistant` option the config and override stuff set on VF's webchat settings dashboard.